Sources/SecureStore/SecureStoreConfiguration.swift

```diff
@@ -3,10 +3,14 @@ import Foundation
 public struct SecureStorageConfiguration {
     let id: String
     let accessControlLevel: AccessControlLevel
+    let localAuthStrings: [String: String]?
 
-    public init(id: String, accessControlLevel: AccessControlLevel) {
+    public init(id: String,
+                accessControlLevel: AccessControlLevel,
+                localAuthStrings: [String: String]? = nil) {
```

Comment:
should this be a struct?
```swift
struct LocalAuthenticationLocalizedString {
    let localizedReason: String
    let localisedFallbackTitle: String
    let localisedCancelTitle: String
}
```

Reply:
Reasonably it could be yeah, happy to change it to being 👍

Reply:
Done 👍